Makefile

```diff
@@ -26,7 +26,8 @@ GO_TEST_FLAGS?=
 
 .PHONY: test
 test:
-	PASSES="unit integration release e2e" ./scripts/test.sh $(GO_TEST_FLAGS)
+	PASSES="unit integration release" ./scripts/test.sh $(GO_TEST_FLAGS)
+	PASSES="e2e" ./scripts/test_e2e.sh $(GO_TEST_FLAGS)
```

Comment:
For the new standalone scripts should we move away from the `PASSES=..` syntax so contributors would just be able to run the script and not need to worry about supplying a parameter?

That could drastically simplify the structure of the scripts possibly.

@serathius what were your thoughts on this as the original issue raiser?

Reply:
I agree, by splitting the `test.sh` we hope to ditch `PASSES` as it's just a custom implementation of targets.

@sharathsivakumar any reason you left `PASSES` support in `./scripts/test_e2e.sh`?

Reply:
@serathius Thanks for the inputs. I left the `PASSES` support to be consistent with the rest of the functions. I also wanted to have this consensus and conversation on ditching `PASSES`  since it will be a significant change. I think I have all the inputs needed to proceed with the changes. I am going to remove the `PASSES` support for `test_e2e.sh` first. Then create separate PRs for the rest of the scripts.